### ISSUE TITLE
Remove or modify unreleased versions from the drop down

### ISSUE THREAD

**Issue by nik9000** [elastic/docs#1480]
We just (#1447) removed non-"live" versions from the version drop down and @rjernst asked a great question:
> Are the in flight versions something that confuse users as well?

I'm no sure to be honest. I *love* that we have `master` and `7.x` in there because it invites the folks curious about the development process to explore it and, hopefully, be delighted about how open we are. But maybe it'd be worth removing them from the list. Or marking them `(unreleased)`. Or *something*.

**Comment by lcawl** [comment]
I think that given that we now have "other versions" in the list (https://github.com/elastic/docs/pull/1481), it's reasonable for users to find 7.x and master there too. In which case, the drop-down list would only contain the supported, released docs.

**Comment by debadair** [comment]
Yes, master and 7.x do cause some confusion--particularly master. Contributors generally understand our versioning model, but regular users don't necessarily make the connection between master and the next major version. 

I do think there's value in surfacing upcoming/in-progress releases, but I worry that adding qualifying text to each entry would be too cluttered--especially at the top of the list. Maybe we could segment the drop-down? Something like:

![dropdown](https://user-images.githubusercontent.com/362578/69763920-8b7d6580-1123-11ea-87d8-3153603dab1a.png)

I don't feel particularly strongly about it--if the consensus is that "Other versions" is sufficient for both future and past versions, that would be ok too.

**Comment by rjernst** [comment]
Perhaps instead of "All version" or "Other versions", we could have "Past versions" and "Unreleased versions". I like the idea of having it still easily findable, but very few people (and mostly just maybe plugin developers) need to find the unreleased version docs.

**Comment by debadair** [comment]
If we added headings for Unrelease and Past versions to the version link farm pages, we could do that. 

I do think that the docs for the next release are of interest to more than just plugin developers--looking at the analytics, the most-viewed pages in 7.5 are the release notes and breaking changes. 
Things get murkier for 7.x and master, and there's significantly more traffic to master than 7.x and 7.5 over the last month, which I suspect reinforces that people are ending up on master when that's not necessarily what they want. The numbers for future versions are tiny compared to current, though.

**Total pageviews 11/1 - 11/27**
Current:  4,035,415 
7.4: 589,331*
7.5: 21,113
7.x: 48,251
Master: 214,576

* The "current" version is actually a separate copy of the current branch, current and 7.4 take you to different pages.

**Comment by gtback** [comment]
[From](https://github.com/elastic/docs/issues/1779#issue-585094395) @rwaight:
> In order to improve navigation on the [Elastic Stack and Product Documentation](https://www.elastic.co/guide/index.html) site, it would be helpful to put `master` into the `other versions` section of the navigation menu.
>
> Currently the `master` version of the documentation is available in the main version menu on the site.  This can cause confusion when reading about capabilities and configurations of currently available products, since the `master` version is preliminary documentation.

**Comment by gtback** [comment]
[From](https://github.com/elastic/docs/issues/1779#issue-585094395) @lcawl:
> Probably you could make the same case for any .x pre-release content (like 7.x branches).

**Comment by gtback** [comment]
[An earlier comment](https://github.com/elastic/docs/issues/1779#issuecomment-621450029) from me: 
>This can be done on a per-book basis by changing the `live:` key in `conf.yaml`. For the ~50% of books that are versioned with the stack (see the [`stacklive` variable](https://github.com/elastic/docs/blob/master/conf.yaml#L60)), this would happen for all of them at once, though in an edge case we could make a one-off exception for a single book by not using that variable.
>
>Alternatively, we could change the order so the "current" book is at the top, followed by older, still-maintained branches, followed by `master`.

**Comment by gtback** [comment]
I agree with @debadair's suggestion above, having 3 sections: 
- Have the current version at the top, in the same section as any other currently-released-and-supported versions. (maybe call this "stable"?)
- Have this followed by a section with the "in development" versions: `master`, `7.8`, and `7.x` as of the time of this comment. ("upcoming" or "future" or "unreleased"?)
- Finally, a way to select "other versions". 

@zuketo @lcawl what do you think?

**Comment by lcawl** [comment]
I think that sounds like a reasonable solution.  I wouldn't call the current release "stable" though, since that "current" URL is always moving (which is part of the reason it gets so many link breakages).  I don't feel strongly for or against any of the other group names, though.

**Comment by gtback** [comment]
By "stable" I meant versions of the software being documented (including the current stable version), not that the documentation itself was stable 😦 . But I understand the potential confusion and will try to think of a better word.

**Comment by rayafratkina** [comment]
The confusion about future options came up a couple of times recently - is there anything we can do to improve this in the short term?
